controllers/nodescalingwatermark_controller.go

```diff
@@ -141,27 +139,26 @@ func (r *NodeScalingWatermarkReconciler) Reconcile(context context.Context, req
 		return r.ManageError(context, instance, err)
 	}
 
-	err = r.UpdateLockedResources(context, instance, []lockedresource.LockedResource{
-		{
-			Unstructured: *obj,
-			ExcludedPaths: []string{
-				".metadata",
-				".status",
-			},
-		},
-	}, []lockedpatch.LockedPatch{})
+	// err = r.UpdateLockedResources(context, instance, []lockedresource.LockedResource{
```

Comment:
Can this be removed?

Reply:
I want to keep this because I'd like to use it. This is what caused the error and I don't know why, I just applied a work around.